### ISSUE TITLE
[Week1/김승규]다음주_계획_건의

### ISSUE THREAD

**Issue by kimsk920825** [kimsk920825/team1_wanted_onboarding#7]
저희 다음주 계획을 아래와 같이 해보는게 어떨까용? (시간은 임의로 정했습니다. 많은 의견 공유 부탁드려욧!)

프로젝트 버전 1: EDA
프로젝트 버전 2: Feature Engineering 
프로젝트 버전 3: 머신러닝 모델링 

우선 첫 목표는 버전 1을 완성시키는 게 목표입니다!
만약 목표한 기한 안에 버전 1을 완성 못 기키거나 팀원 중 힘든 분이 있으시다면 서로 도우면서 버전 1을 완성시키는 것이 프로젝트의 주된 목표라고 개인적으로 생각합니다!
 
계획 1. 
화요일 (목표: 프로젝트 버전 1 완성. 발표자 추첨하기)
: 13:00까지 개인적으로 EDA 한 파일을 Issue에 업로드한 다음 서로의 insight 공유하기.
: 만약 EDA 작업이 부족했다면 추가 EDA 작업을 하고 수집 후 정리 [수집하고 정리는 제가 하겠습니다! 다른 분이 하고 싶으시면 말씀해 주세욧!!]

목요일
: 목요일까지 Feature Engineering 프로젝트 버전 2 완성. (정확한 시간은 화요일 이후 정함)

금요일
: 금요일까지 머신러닝 모델링 프로젝트 버전 3완성. (정확한 시간은 화요일 이후 정함)
: 발표자 추천

계획 2.
화요일
: 13:00까지 개인적으로 EDA 한 파일을 Issue에 업로드한 다음 서로의 insight 공유하기.
수요일
: 만약 EDA 작업이 부족했다면 추가 EDA 작업을 하고 수집 후 정리 
목요일 - 금요일
: 이해가 힘든 팀원이 있다면 서로 도와주고 이해시켜주기
: 발표자 추천 후  발표 연습.

**Comment by leeyunji964** [comment]
저는 좋은 것 같습니다!

**Comment by rjsdl56** [comment]
저도 좋습니다!!

**Comment by Lucia-KIM** [comment]
고생 많으십니다~ 저도 일정에 이견없습니다! 
혹시 화요일에는 저희 팀끼리 구글 밋 등으로 모여서 한번 방향성 이야기 해보면 어떨까요? 
텍스트로 의견을 교환하는데 한계가 있을거 같아서요

**Comment by cocong2** [comment]
화요일 오후 1시까지 EDA 공유 좋습니다!
다만 제가 재직중이라 오후 1시에는 공유 시간을 가지기 어려울 것 같아요..

**Comment by nahye9** [comment]
저도 좋습니다. 계획 정리 해주셔서 감사합니다.

**Comment by Lucia-KIM** [comment]
> 화요일 오후 1시까지 EDA 공유 좋습니다!
> 다만 제가 재직중이라 오후 1시에는 공유 시간을 가지기 어려울 것 같아요..

네 말씀감사합니다! 저녁 일정으로 조율해도 좋고, 다른분들 의견에 꼭 필요치 않다고 보이시면 꼭 진행하지 않아도 좋습니다~

**Comment by kimsk920825** [comment]
퇴근 하고 댓글 하나 하나 읽어보고 의견 정리해보겠습니닷!!!
